addons/mrp/report/mrp_report_mo_overview.py

```diff
@@ -543,6 +542,7 @@ def _get_replenishment_lines(self, production, move_raw, replenish_data, level,
             else:
                 replenishment['summary']['receipt'] = self._check_planned_start(production.date_start, self._get_replenishment_receipt(doc_in, replenishment.get('components', [])))
             replenishment['summary']['mo_cost_decorator'] = self._get_comparison_decorator(replenishment['summary']['real_cost'], replenishment['summary']['mo_cost'], replenishment['summary']['currency'].rounding)
+            replenishment['summary']['formatted_state'] = doc_in._name == 'mrp.production' and self._format_state(doc_in, replenishment['components']) or self._format_state(doc_in)
```

Comment:
```suggestion
            replenishment['summary']['formatted_state'] = self._format_state(doc_in, replenishment['components']) if doc_in._name == 'mrp.production' else self._format_state(doc_in)
```
I find it slightly more readable, but it's up to you.

Reply:
you're the one who'll probably be reading it the most :wink: (I changed it)